crates/frontend/src/pages/dimensions.rs

```
@@ -98,12 +99,12 @@ pub fn dimensions() -> impl IntoView {
             let schema = row["schema"].clone().to_string();
             let schema = serde_json::from_str::<Value>(&schema).unwrap_or(Value::Null);
 
-            let function_name = row["function_name"].to_string();
-            let fun_name = match function_name.as_str() {
-                "null" => None,
-                _ => Some(json!(function_name.replace('"', ""))),
-            };
-
+            let validation_function_name = row
+                .get("function_name")
+                .map(|v| v.as_str().unwrap_or_default().to_string());
```

why unwrap_or_default here , cant we flatten the option here?
same in below line

---

No my point was in case of v.as_str() , lets not do default for that , lets convert it to option only, 
converting to default empty string can be misleading in some weird case

---

It will always be string, there is no doubt in that. That is enforced by API and DB. The frontend should not be rendering anything other than string.